scripts/generate_provenance

```diff
@@ -28,19 +35,22 @@ cp build.toml tmp/build.toml
 )
 
 # Build the binary and generate the provenance file.
-# `|| true` is needed so that we can continue to the rest of the script, particularly the cleanup part.
+# `|| true` is needed so that we can continue to the rest of the script,
+# particularly the cleanup part.
 ./tmp/transparent-release -build_config_path=tmp/build.toml -git_root_dir=. || true
 
 
 # cleanup
 rm -rf tmp
 
-# On GitHub we need to store the generated provenance files in a location that can easily be copied
-# to the provenance branch. We pass an optional argument to the script to trigger this behaviour.
-if [ $# -gt 0 ]; then
+# On GitHub we need to store the generated provenance files in a location that
+# can easily be copied to the provenance branch. We pass an optional argument
+# to the script to trigger this behaviour.
```

Comment:
Not sure I understand this -- what is the optional argument?

Reply:
Fixed.